keyboards/pizzakeyboards/slice65/mcuconf.h

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2022 Christoph Baumann
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#pragma once
+
+#include_next <mcuconf.h>
+
+#undef STM32_PREDIV_VALUE
+#define STM32_PREDIV_VALUE 2
```

Comment:
```suggestion
#define STM32_PREDIV_VALUE 2

```

Reply:
What's the purpose of the blank line?

Reply:
[quick-ish explaination]( https://medium.com/@callmeryan/no-newline-at-end-of-file-navigating-gits-warning-for-android-developers-af14e73dd804)

Reply:
Github makes this look like a blank line, but I see now that this is what vim would do by default.  I've just never saved this file.  Will correct.